crates/bevy_winit/src/winit_windows.rs

```diff
@@ -98,6 +98,55 @@ impl WinitWindows {
             .with_transparent(window.transparent)
             .with_visible(window.visible);
 
+        match &window.name {
+            None => {}
```

Comment:
Sorry for nitpicking but shouldn't this be an if let given the empty None case?

Reply:
Has this match statement achieved the goal?

Reply:
Thanks! Applied the suggestion. I thought I had three match branches earlier...